test/unit/price-formatting/parse-spec.js

```diff
@@ -69,6 +69,10 @@ describe('price-formatting parse', () => {
         testConversion(0, 5, priceFormatOptions.ModernFractions, 1);
 
         testConversion(0, 5, priceFormatOptions.ModernFractions, 2);
+
+        testConversion(-0.5625, 5, priceFormatOptions.ModernFractions);
+
+        testConversion(-17.25, 5, priceFormatOptions.ModernFractions);

```

Comment:
Nitpick. 

We might get additional coverage for free by duplicating previous cases with negative value as input:
testConversion(2567.90625, 5, priceFormatOptions.ModernFractions);
```
        testConversion(-2567.21875, 5, priceFormatOptions.ModernFractions);
        testConversion(-2567.21875, 5, priceFormatOptions.ModernFractions, 1);
        testConversion(-2567.21875, 5, priceFormatOptions.ModernFractions, 2);
        testConversion(-2567.90625, 5, priceFormatOptions.ModernFractions, 1);
        testConversion(-2567.90625, 5, priceFormatOptions.ModernFractions, 2);
        testConversion(-2567.9921875, 5, priceFormatOptions.ModernFractions, 2);
```

Reply:
Done